.github/workflows/CI.yml

```diff
@@ -126,13 +126,13 @@ jobs:
         run: $SBT ratCheck || (cat target/rat.txt; exit 1)
 
   build-test-package:
-    name: 'Build, Test, and Package (OS: ${{ matrix.os }}, Node: ${{ matrix.node }}, Java: ${{ matrix.java_version }})'
+    name: 'Build, Test, and Package (OS: ${{ matrix.os }}, Node: ${{ matrix.node }}, Java: ${{ matrix.java_version }}, VS Code: ${{ matrix.vscode }})'
     strategy:
       matrix:
         java_distribution: [ temurin ]
         java_version: [ 8, 11, 17 ]
         os: [ macos-12, ubuntu-20.04, windows-2019 ]
-        node: [ '16', '18.20.1' ] # version 18.20.2 (current latest of 18) is broken on windows
```

Comment:
Add this back, nodejs 16 is our minimal supported version so we want to make sure to test it in CI. Also add 20 into this file as well